### DISCUSSION TITLE
OTA epg options

### DISCUSSION THREAD

**Discussion by techiegirl2day** [OpenSageTV/Community#13] [Guide/EPG]
My Schedules direct is about to expire. I was planning to take a hiatus from recording for the summer, so want to delay any renewal of service until I plan to start regular taping again.  I have one show The Blacklist that wont finish up for a month. What can I do to make sure the listings populate so can receive all the upcoming episodes till it's finale? I have never tried using just the OTA data info. How will this work or should I just bit the bullet & renew?  Was hoping to have a later renewal anniversary date than current one so in future, date will fall after regular season ends for most shows. I know that is changing it up with "Summer season" offerings. Suggestions?

Galaxysurfer

**Comment by wayner9** [comment]
Can you just do timed recordings?

**Reply by techiegirl2day** [reply]
I guess.  I would have to watch schedule from other tv listings to make sure I don't miss anything. Was hoping for a less labour intensive option that doesn't need babysitting. It would work as long at Day & time don't change from norm. Weaning self off tv addiction isn't easy.  lol. SageTV makes it so easy.  I was using no tv guide to be excuse to step away. How does ota get listings into SageTV if not running schedulesdirect?

**Reply by JustFred51** [reply]
Each OTA station broadcasts it's own guide (typically only 2-3 hours worth) within the data stream.  Probably good enough for what you're trying to accomplish.

**Reply by emveepee** [reply]
Paying for two months is probably your best option.   You can try OTA and  if you use the HDHR you can get their 24hr XMLTV but how much time do you want to spend now and later to reset things to save $9

**Reply by jzhvymetal** [reply]
If you have a docker server you could run zap2xml that will work with the SageTV XMLTV plugin.  Altough working I still use SD as it is stable.
